tools/profile_apply

```diff
@@ -250,12 +250,23 @@ def apply(tabs):
         for chart_rows in split_by_title(rows):
             put_chart(chart_rows)
 
+    def clear_form(form_id):
+        """Removes all the form_fields from the given form."""
+        db.execute('update form_field set parent_form_field = null'
+                   ' where form_id = %s', form_id)  # remove foreign keys
+        db.execute('delete from form_field where form_id = %s', form_id)
+
     def apply_chart(rows, form_id):
         """Applies the chart definition given rows from the chart tab."""
+        clear_form(form_id)
         section = None
         sections = []
         section_rows = []
+
         for row in rows:
+            if row['title'].strip():
+                sections.append(('[chart_divider]',
```

Comment:
* No each chart should have it's "chart" section. It would work without it (and so do forms), but only if you have them one after the other.
* Yes, the title is the real divider, not the "chart" (as it is to forms).
* Yes